patches/lua-tree.patch

```diff
@@ -184,7 +226,26 @@ index 625bb0c63..3c7126165 100644
  local tostring     = tostring
  local concat       = table.concat
  local insert       = table.insert
-@@ -326,7 +322,7 @@ Schema.validators = {
+@@ -43,6 +40,18 @@ local _cache = {}
+ local _workspaceable = {}
+ 
+ 
++local new_tab
++do
++  local ok
++  ok, new_tab = pcall(require, "table.new")
++  if not ok then
++    new_tab = function(narr, nrec)
++      return {}
++    end
++  end
++end
++
++
```

Comment:
This localized function makes sense and it is only used if a field is referenceable. Maybe in the patch comments you can indicate a little more under `Reintroducing fallback Lua functions, as the OpenResty tooling is not available for use`. Also could we add a small blurb as to why we are adding this function in the code?

```
-- 'table.new' is an OpenResty function and is not available to goks
local new_tab
```

something like that; please rephrase to make more coherent.

Reply:
I can be a bit more specific within the patch file as [mentioned here](https://github.com/Kong/goks/pull/27#discussion_r958509263). Was there anything specific you wanted to call out other than what functions have been added back & to what files?

As for adding in comments into the code itself, sure thing.

Reply:
I think just listing out the OpenResty functionality that you are re-implementing is good enough; just something to catch our eyes and help the next person looking at this.